packages/edit-widgets/src/store/actions.js

```diff
@@ -113,6 +118,17 @@ export function* saveWidgetArea( widgetAreaId ) {
 		return true;
 	} );
 
+	// Get all widgets that have been deleted
+	const deletedWidgets = areaWidgets.filter( ( widget ) => {
+		const { id } = widget;
```

Comment:
Nit: Could remove this line and do the destructuring in the function's arguments.